textdb/textdb-dataflow/src/main/java/com/google/re2j/PublicParser.java

```diff
@@ -0,0 +1,21 @@
+package com.google.re2j;
+
+/**
+ * Wrapper class of re2j.Parser.<!-- --> This class parses a regex string to a PublicRegexp object. <br>
```

Comment:
And Javadoc will generate HTML exactly according to the comment. So I have to add "<br>" to have a new line.